_blog/posts/2017/0.1-announcement.md

```diff
@@ -36,18 +36,19 @@ Google, IBM and Lyft joined forces to create Istio from a desire to provide a re
 
 **Fleet-wide Visibility**: Failures happen, and operators need tools to stay on top of the health of clusters and their graphs of microservices. Istio produces detailed monitoring data about application and network behaviors that is rendered using [Prometheus](https://prometheus.io/) & [Grafana](https://github.com/grafana/grafana), and can be easily extended to send metrics and logs to any collection, aggregation and querying system. Istio enables analysis of performance hotspots and diagnosis of distributed failure modes with [Zipkin](https://github.com/openzipkin/zipkin) tracing.
 
-<figure>
-    <a href="{{home}}/img/istio_grafana_dashboard-new.png">
-    <img style="max-width: 100%;" src="{{home}}/img/istio_grafana_dashboard-new.png" alt="Grafana Dashboard with Response Size" title="Grafana Dashboard with 
-    Response Size" />
-    </a>
-</figure>
-
-<figure>
-    <a href="{{home}}/img/istio_zipkin_dashboard.png">
-    <img style="max-width: 100%;" src="{{home}}/img/istio_zipkin_dashboard.png" alt="Zipkin Dashboard" title="Zipkin Dashboard" />
-    </a>
-</figure>
+{% include figure.html width='100%' ratio='55.42%'
+    img='./img/istio_grafana_dashboard-new.png'
+    alt='Grafana Dashboard with Response Size'
+    title='Grafana Dashboard with Response Size'
+    caption='Grafana Dashboard with Response Size'
+    %}
+
+{% include figure.html width='100%' ratio='29.91%'
```

Comment:
those numbers, 29.91% - where are they coming from ?

Reply:
It's (image height / image width) * 100. I mentioned this in the updated instructions on how to insert an image.

With figure.html, the author specifies the relative width of the image as compared to the surrounding content. The author also specifies the aspect ratio of the image. This is enough info to be able to calculate the actual width & height of the image during layout.

Reply:
can’t a computer calculate that ? preserving the aspect ratio should be the default, this seems error prone (and cumbersome)?